src/main/java/lotto/collections/Money.java

```diff
@@ -0,0 +1,14 @@
+package lotto.collections;
+
+public class Money {
+
+  private int money;
```

Comment:
오, 좋은 피드백입니다. Immutable Object라는 키워드를 처음 알게 되었네요.
돈에 관련된 클래스는 생성 시점을 제외하고 변경 가능성이 없고 불변해야 하므로, 변경 가능한 범위가 없도록 해야 하는 것이 맞겠습니다. 또한 상속도 하지 않는 것이 일단은 좋겠네요. (만약에 달러, 원화, 위안화 등으로 여러 화폐를 지원해야 한다고 하면 인터페이스를 도입하는 것이 좋겠구요)

사실 로또를 구매하는 PurchaseService에서는 Money를 불변 객체로 취급하여 받고 있습니다.
```
  public LottoTickets purchase(final Money money) {
```

Reply:
```
The final keyword when used for parameters/variables in Java marks the reference as final. In case of passing an object to another method, the system creates a copy of the reference variable and passes it to the method. By marking the new references final, you protect them from reassignment.
```

* https://stackoverflow.com/questions/4162531/making-java-method-arguments-as-final/4162782